RIRImageResizing/Classes/NSObject/RIRResizeImageOperationParameters.h

```diff
@@ -0,0 +1,43 @@
+//
+//  RIRResizeImageOperationParameters.h
+//  Pods
+//
+//  Created by Richard Reitzfeld on 11/21/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+
+
+
+
+typedef NS_ENUM(NSInteger, UIImage_RIRResizing_ResizeMode) {
+    UIImage_RIRResizing_ResizeMode_ScaleToFill,
+    UIImage_RIRResizing_ResizeMode_AspectFit,
+    UIImage_RIRResizing_ResizeMode_AspectFill,
+    
+    UIImage_RIRResizing_ResizeMode_first = UIImage_RIRResizing_ResizeMode_ScaleToFill,
```

Comment:
first and last need two underscores , i.e. `UIImage_RIRResizing_ResizeMode__first` and `UIImage_RIRResizing_ResizeMode__last`.